client/src/components/Reports/index.js

```diff
@@ -0,0 +1,358 @@
+                        <Popper
+                          anchorEl={this.popperNode}
+                          open={Boolean(options.children)}
+                        >
+                          <Paper
+                            square
+                            {...options.containerProps}
+                            style={{
+                              width: this.popperNode
+                                ? this.popperNode.clientWidth
+                                : null
+                            }}
+                          >
+                            {options.children}
+                          </Paper>
+                        </Popper>
+                      )}
+                    />
+                  </div>
+                </div>
+                <div className={classes.block}>
+                  <div className={classes.stockInfo}>
+                    <Typography variant="h6">
+                      Price: ${data.length ? data[data.length - 1].close : ""}
+                    </Typography>
+                    <Typography variant="h6" style={{ marginLeft: "50px" }}>
+                      Change:{" "}
+                      {data.length
+                        ? this.changePoints(
+                            data[data.length - 1].close,
+                            data[data.length - 1].open
+                          )
+                        : ""}
+                    </Typography>
+                  </div>
+                  <div className={classes.stockInfo}>
+                    <Typography variant="h6">
+                      Volume:{" "}
+                      {data.length
+                        ? this.shortenVolume(data[data.length - 1].volume)
+                        : ""}
+                    </Typography>
+                    <Typography variant="h6" style={{ marginLeft: "50px" }}>
+                      Change %:{" "}
+                      {data.length
+                        ? this.changePercent(
+                            data[data.length - 1].close,
+                            data[data.length - 1].open
+                          )
+                        : ""}
+                    </Typography>
+                  </div>
+                </div>
+              </Grid>
+              <Grid item xs={12} md={12}>
+                <Paper
+                  className={classes.paper}
+                  style={{ position: "relative" }}
+                >
+                  <AppBar position="static">
+                    <Tabs
+                      value={value ? value : 0}
+                      onChange={this.handleTabChange}
+                      indicatorColor="primary"
+                      textColor="secondary"
+                      variant="scrollable"
+                      scrollButtons="auto"
+                    >
+                      <Tab label="Price" />
+                      <Tab label="Moving Average" />
+                      <Tab label="Moving Average Convergence" />
```

Comment:
Clicking on this actually brings up an error message. Also if we are suppose to see the chart. I am not sure if it's there. And maybe its a loading issue. We may need to use Async Await here.